nyc_taxi/dashboard.ipynb

```diff
@@ -11,12 +11,12 @@
+    "<img align='right' style='padding:0px 0px 0px 0px;' \n",
+    "src='./thumbnails/index.png'/>\n",
     "\n",
     "The [NYC Taxi trips](https://anaconda.org/jbednar/nyc_taxi/notebook) dataset is a well-studied data science example. Here we show how to build a simple dashboard for exploring 10 million taxi trips in a Jupyter notebook using [Datashader](http://datashader.org), then deploying it as a standalone dashboard using [Panel](http://panel.pyviz.org).\n",
```

Comment:
- I don't think there's a need to link to this old notebook.
- Updating old links is useful, I'll add that to the checklist.

```suggestion
    "The NYC Taxi trips dataset is a well-studied data science example. Here we show how to build a simple dashboard for exploring 10 million taxi trips in a Jupyter notebook using [Datashader](https://datashader.org), then deploying it as a standalone dashboard using [Panel](https://panel.holoviz.org).\n",
```